app/(candidate)/dashboard/components/tasks/flows/TaskFlow.js

```diff
@@ -231,6 +252,24 @@ export default function TaskFlow({
         onConfirm={handleCloseConfirm}
       />
       <Modal open={open} closeCallback={handleClose}>
+        {phoneListToken && (
+          <LongPoll
+            {...{
+              pollingMethod: async () => {
+                return await getP2pPhoneListStatus(phoneListToken)
+              },
+              onSuccess: (result) => {
+                const { phoneListId, leadsLoaded } = result || {}
+                handleChange({
+                  phoneListId,
+                  leadsLoaded,
+                })
+                setStopPolling(true)
+              },
+              stopPolling,
```

Comment:
What about adding `limit: 30` (or 60) so that it doesn't use the default limit of 0 and potentially continue indefinitely?